test/integration/TestEnvironments.ts

```diff
@@ -52,36 +60,38 @@ export class TestEnvironment {
     }
 
     async getClientWithIdentity(identity): Promise<SocialClient> {
-        const client = await loginWithIdentity(this.synapseContainer.getAddress(), identity)
+        const client = await loginWithIdentity(
+            this.synapseContainer.getAddress(),
+            identity
+        )
         this.clients.push(client)
         return client
     }
 
     createUserOnServer(identity): Promise<SocialId> {
         return createUser(this.synapseContainer.getAddress(), identity)
     }
-
 }
 
 /**
  * This is an easy way to load a test environment into a test suite
  */
 export function loadTestEnvironment(): TestEnvironment {
+    ;(global as any).localStorage = new LocalStorage('.storage')
```

Comment:
I think the `;` is a typo, isn't it?

Reply:
No, that's just the linter preventing errors from starting a line with parenthesis since js doesn't need the `;` at the end of the line